packages/common/src/converter/data-converter.ts

```diff
@@ -182,8 +188,21 @@ export function mapToPayloadsSync<K extends string>(
   ) as Record<K, Payload>;
 }
 
-export const defaultDataConverter = new CompositeDataConverter(
-  new UndefinedPayloadConverter(),
-  new BinaryPayloadConverter(),
-  new JsonPayloadConverter()
-);
+export interface DefaultDataConverterOptions {
+  protobufClasses?: Record<string, unknown>;
+}
+
+export class DefaultDataConverter extends CompositeDataConverter {
+  constructor({ protobufClasses }: DefaultDataConverterOptions = {}) {
```

Comment:
I will have to dig into protobufjs, but all other proto impls I am aware of maintain a global type registry that is required for resolving `google.protobuf.Any`.

Reply:
Well, I can't find this, nevermind.